packages/client/hmi-client/src/components/operator/tera-operator-menu.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<nav>
+		<Button
+			type="button"
+			size="small"
+			@click="showMenu"
+			icon="pi pi-plus"
+			aria-haspopup="true"
+			aria-controls="overlay_menu"
+			severity="secondary"
+			:style="style"
+		/>
+		<Menu ref="menu" id="overlay_menu" :model="menuItems" :popup="true" />
+	</nav>
+</template>
+<script setup lang="ts">
+import { ref } from 'vue';
+import type { WorkflowNode } from '@/types/workflow';
+import Menu from 'primevue/menu';
+import Button from 'primevue/button';
+
+defineProps<{
+	node: WorkflowNode<any> | null;
+	style: { top: string; left: string };
```

Comment:
Again, the style should be done on the parent component, there is not reason to have those here as props. Or I misunderstanding something.

Reply:
`tera-canvas-item` has the same issue too